docs/how-the-agent-works.asciidoc

```diff
@@ -26,6 +26,40 @@ Examples of this are
 Framework integrations require some limited code changes in your app.
 E.g. for Django, you need to add `elasticapm.contrib.django` to `INSTALLED_APPS`.
 
+[float]
+[[how-it-works-no-framework]]
+==== What if you are not using a framework
+
+In case you are not using a supported framework, for example a simple Python script, you can still
+leverage the <<automatic-instrumentation,Automatic Instrumentation>>.
+
+In order to collect the Spans generated by the supported libraries, you will need to create a Transaction.
```

Comment:
Can we add a note here about calling `elasticapm.instrument()` in addition to creating a transaction? That's a key requirement for actually getting spans from instrumented libraries.

Reply:
Hello @basepi ! I've added the requirement to invoke `elasticapm.instrument()` just once.
I am not 100% sure of the wording used.